### ISSUE TITLE
[DataTable] - Moving md-table-column--grow class from first column to last column

### ISSUE THREAD

**Issue by artemukolov** [mlaursen/react-md#271]
### Moving md-table-column--grow class from first column to last

I added DataTable with TableHeader to my page. I didn't add autoAdjust to TableRow and didn't add prevent-grow class to some column in TableHeader

![](http://ukolov.info/github/03032017_3.png)

Also I use react-router in my app. And then I open page with DataTable in first time I see something like this

![](http://ukolov.info/github/03032017_1.png)

Then I move to another page and return to page with DataTable I see it

![](http://ukolov.info/github/03032017_2.png)

I notice what class md-table-column--grow is moved from first column to last column. This bug don't reproduced, if DataTable has just header without any rows.

- React 15.4.2
- React-MD 1.0.5

**Comment by mlaursen** [comment]
In this case, you would want to update your code to be this:

```js
<TableHeader>
  <TableRow autoAdjust={false}>
    <TableColumn>File Name</TableColumn>
    <TableColumn>File Type</TableColumn>
    <TableColumn>Last Modified</TableColumn>
  </TableRow>
</TableColumn>
```

The `autoAdjust` is defaulted to `true`. But in the next version of react-md, this auto adjust functionality will be removed. See #252

**Comment by artemukolov** [comment]
And what I must do now? That's weird behaviour. AmI right?

**Comment by mlaursen** [comment]
This should be fixed now in 1.1.0
